dominion/frontend/src/component/Footer/Partners.js

```diff
@@ -1,15 +1,64 @@
 import React from 'react';
-import { Grid, Typography } from '@material-ui/core';
+import PropTypes from 'prop-types';
```

Comment:
Any reason Partners is part of Footer? Isn't it part of the page and hence should be its own component i.e. `src/components/Partners.js`

If it **is** part of Footer, then Footer should include Partners and pages should only include Footer i.e. Partners shouldn't be used outside the Footer.